doc/How-Tos/DependencyInjection/DependencyInjection-setup.md

```diff
@@ -0,0 +1,88 @@
+# How-To: Use Services with Dependency Injection
+
+Dependency injection (DI) is an important design pattern for building loosely-coupled software that allows for maintainability and testing. This tutorial will walk you through how to register services with a service provider for use throughout your application.
+
+NOTE: This guide assumes you used the Uno.Extensions `dotnet new` template to create the solution. You can verify this by looking for a generated `App.xaml.host.cs` file.
```

Comment:
```suggestion
> [!Tip] This guide assumes you used the Uno.Extensions `dotnet new unoapp-extensions` template to create the solution. You can verify this by looking for a generated `App.xaml.host.cs` file.
```

When you say generated, does it mean that it was generated when creating the template?

Reply:
Yeh, the app.xaml.host.cs file is generated by the template. Change "for a generated" to "for the generated"
Also, suggest we link to the getting started doc here in case they need to walk through creating the app from the template